src/Forms/DatetimeField.php

```diff
@@ -189,141 +375,167 @@ public function setValue($value, $data = null)
         // Save value
         $this->value = $value;
 
-        // Shift iso date into timezone before assignment to subfields
-        $timezoneFormatter = $this->getTimezoneFormatter();
-        if ($timezoneFormatter) {
-            $value = $timezoneFormatter->format($timestamp);
-        }
-
-        // Set date / time components, which are unaware of their timezone
-        list($date, $time) = explode(' ', $value);
-        $this->dateField->setValue($date, $data);
-        $this->timeField->setValue($time, $data);
         return $this;
     }
 
-    /**
-     * localised time value
-     *
-     * @return string
-     */
     public function Value()
     {
-        $date = $this->dateField->Value();
-        $time = $this->timeField->Value();
-        return $this->joinDateTime($date, $time);
+        return $this->iso8601ToLocalised($this->value);
+    }
+
+    public function dataValue()
+    {
+        return $this->iso8601ToDataValue($this->value);
```

Comment:
`$this->value` should only be saved in ISO8601. The `iso8601ToDataValue` method should not be necessary. Instead, we ensure that we only assign `$this->value = $some8601ConvertedVal`.